rpi5.yaml

```diff
@@ -0,0 +1,231 @@
+desc: "Raspberry 5 with xen dom0less"
+min_ver: "0.22"
+
+
+variables:
+  # Common build configuration
+
+  YOCTOS_WORK_DIR: "yocto"
+
+  DOM0_OS: "zephyr"
+
+  # Raspberry 5 build configuration
+
+  RPI_MACHINE: "raspberrypi5"
+  RPI_SOC_FAMILY: "bcm2712"
+
+  ZEPHYR_DOM0_DIR: "zephyr"
+  ZEPHYR_DOM0_MACHINE: "rpi_5_xen_dom0"
+  ZEPHYR_DOM0_TARGET: "zephyr/samples/subsys/shell/shell_module"
+  ZEPHYR_DOM0_BUILD_DIR: "build-dom0"
+
+  DOMD_BUILD_DIR: "build-domd"
+  XT_XEN_DTB_NAME: "%{RPI_SOC_FAMILY}-%{RPI_MACHINE}-xen.dtb"
+  XT_DOMD_DTB_NAME: "%{RPI_SOC_FAMILY}-%{RPI_MACHINE}-domd.dtb"
+
+  XT_KERNEL_BRANCH: "rpi-6.6.y"
+  XT_KERNEL_REV: "${AUTOREV}"
+
+  XT_DOMD_IMAGE: "image"
+
+common_data:
+  # Sources used by domd
+  domd_sources: &DOMD_DOMU_SOURCES
+    - type: git
+      url: "git://git.yoctoproject.org/poky"
+      rev: "scarthgap"
+    - type: git
+      url: "git://git.openembedded.org/meta-openembedded"
+      rev: "scarthgap"
+    - type: git
+      url: "git://git.yoctoproject.org/meta-virtualization"
+      rev: "scarthgap"
+    - type: git
+      url: "git://github.com/dsemenets/meta-xt-common.git"
```

Comment:
@lorc @rshym As can be seen we're using scarthgap release to built rpi5 domd image. According to the discussions to merge this branch we have to get rid from personal forks. So the main topic is how we should merge scarthgap to meta-xt-common. My suggestion is to merge scarthgap section int meta-xt-common to the "scarthgap" branch. But there were compains against it. This comment is intended to start discussion about the correct way to merge it to meta-xt-common

Reply:
We have to produce some kind of "roadmap" to integrate new yocto release into existing infrastructure.

Reply:
@GrygiriiS

Reply:
We already have the defined policy for the upgrade of yocto, as we have discussed during the switch to kirkstone.

1. We make developer's branches with scarthgap for xt-common, xt-rcar.
2. We switch devel3 or devel4 to scarthgap on developer's branches.
3. We locally build and test product built with scarthgap on developer's branches for xt-common, xt-rcar, product.
4. We make branches with kirkstone for xt-common, xt-rcar.
5. We switch ALL products to kirkstone branches. devel3, devel4, aos3.
6. We merge scarthgap developer's branches into xen-troops main for xt-common, xt-rcar.
7. We upgrade release for xt-core (common and rcar).
8. Products with already implemented support for the scarthgap are upgraded to scarthgap on the main branch.

Reply:
@lorc could you please comment on my last statement regarding the roadmap to switch yocto release?

Seems we need to put it into some document, for future, with some changes, if required.

Reply:
Well, I am not sure we should be blocked by meta-xt-prod-devel-rcar. RPI5 project uses only meta-xt-common. We can move meta-xt-common to experimental scarthgap support and see what would break.

Reply:
Currently we doesn't depend on meta-xt-common